HOTFIX Remove HTTP protocol coercion

mwbenowitz opened the pull request:
HOTFIX - No JIRA ticket

### This PR does the following:
The `formatUrl` shim was put in to force URLs to use a certain protocol for environmental reasons (generally being forced to https). Due to updates in the environments this is no longer necessary.

The method is still in place but now simply adds a protocol to any URL that is missing one (this is necessary, for example, with covers). This should allow for certain components, such as the reader app, to work as intended in all environments.

This also adds a new environment: `local`. This is similar to `development` but has some specific settings to allow for easier local development. It can be started by using `npm run local-start` and requires that a local instance of the webpub-reader app be running at `localhost:4444`

### Testing requirements & instructions: 
- Run a local instance of the app with `npm run local-start`
- Verify that all resources load properly with a focus on covers, the ePub reader and other 3rd party readers embedded via iFrame

hokei commented on src/app/components/Card/EditionCard.jsx:
```diff
@@ -168,9 +168,9 @@ export default class EditionCard {
+    const base64BookUrl = Buffer.from(formatUrl(url)).toString('base64');
     const encodedBookUrl = encodeURIComponent(`${base64BookUrl}`);
-
+    console.log(eReaderUrl)
```
console.log

mwbenowitz replied:
Thanks!

hokei commented:
is the idea that running `local` should point you to the local copy of webpub-viewer?  (development still points to the local copy)

mwbenowitz commented:
Yeah, I think I fixed the `development` / `local` issue in webpack in the last push I just made. But yes, they should be the same with the exception of which webpub-viewer is being used. I also removed the GA tracking code from local so at least local test sessions won't show up as users.
